content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
can you convert the existing tracing task into a subsection thingy that has two tasks (with Zipkin, with Lightstep) ?
otherwise, this shows up as yet another task in the main index of tasks under networking

Reply:
@geeknoid @rshriram we need some sort of consistent policy for vendor integration and documentation here, I think. Is our plan to provided "With X" sections for all policy/telemetry tasks? Or should we have an "Integrations" tab for tags for customizations, etc. ?